tests/epyccel/recognised_functions/test_numpy_types.py

```diff
@@ -112,22 +112,22 @@ def add_numpy_to_numpy_type(np_s_l, np_s_r):
 
     pyccel_result = epyccel_func(integer64, fl32)
     python_result = add_numpy_to_numpy_type(integer64, fl32)
-    assert pyccel_result == python_result
+    assert np.isclose(pyccel_result, python_result)
     assert isinstance(pyccel_result, type(python_result))
 
     pyccel_result = epyccel_func(integer64, fl64)
     python_result = add_numpy_to_numpy_type(integer64, fl64)
-    assert pyccel_result == python_result
+    assert np.isclose(pyccel_result, python_result)
     assert isinstance(pyccel_result, type(python_result))
 
     pyccel_result = epyccel_func(fl64, complex64)
     python_result = add_numpy_to_numpy_type(fl64, complex64)
-    assert pyccel_result == python_result
+    assert np.isclose(pyccel_result, python_result)
     assert isinstance(pyccel_result, type(python_result))
 
     pyccel_result = epyccel_func(complex128, fl64)
     python_result = add_numpy_to_numpy_type(complex128, fl64)
-    assert pyccel_result == python_result
+    assert np.isclose(pyccel_result, python_result)
     assert isinstance(pyccel_result, type(python_result))
```

Comment:
Be careful! You should not use `numpy.isclose`, but rather `numpy.allclose` (see [docs](https://numpy.org/doc/stable/reference/generated/numpy.allclose.html#numpy.allclose)).

Further you should provide relative and absolute tolerances (`RTOL`, `ATOL`) which can be defined at the beginning of the test file. Using a value of `1e-14` for both should be adequate. See [this file](https://github.com/pyccel/pyccel/blob/master/tests/epyccel/test_epyccel_mod.py) as an example.

Reply:
Sorry, when comparing scalar values `isclose` is equivalent to `allclose`, so they can both be used here.

Reply:
Thank you! I have updated my PR to include the relative and absolute tolerances (RTOL, ATOL).